schemas/com.vimeo/interaction/jsonschema/1-0-0

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo interactive video hotspot/panel click.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "interaction",
```

Comment:
Maybe rename to `interaction_event` or `interactive_click_event`?

Reply:
Clarified [here](https://github.com/snowplow/iglu-central/pull/1282#discussion_r1219939228) - this is used as a context entity.